WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -499,11 +526,8 @@ protected void onCreate(Bundle savedInstanceState) {
             mShowGutenbergEditor = savedInstanceState.getBoolean(STATE_KEY_GUTENERG_IS_SHOWN);
         }
 
-        // Ensure we have a valid post
-        if (mPost == null) {
-            showErrorAndFinish(R.string.post_not_found);
-            return;
-        }
+        // ok now we are sure to have a valid Post, let's start the editing session tracker
```

Comment:
Cave we move this immediately after the check for `mPost` and `mSite != null` here: https://github.com/wordpress-mobile/WordPress-Android/pull/9293/files#diff-38b6aea9cfff5f3e0e4b7389b44de602R508

Reply:
ah scratch that, I see that `mShowGutenbergEditor` is needed. Maybe just add that info to the comment.

Reply:
I addressed it in c8e9472b3c by changing the method signature and receiving all parameters, so it's easier to see what the method needs instead of looking into it to see which class members it's using in there 👍